contentwarehouse/snippets/create_document_sample.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START contentwarehouse_create_document]
+
+from typing import Optional
+
+from google.cloud import contentwarehouse
+
+
+def sample_create_document(
+    project_number: str,
+    location: str,
+    raw_document_path: str,
+    raw_document_file_type: int,
```

Comment:
Change this back to use the RawDocument Type Enum or just hardcode the enum in where it is used later in the sample (with the link to the documentation)